scripts/csv/data.csv

```diff
@@ -0,0 +1,9 @@
+﻿Non-linear eden allocation

```

Comment:
Is it fine to include such data on csv file?

Reply:
Yep, it is just a title. It is being taken into account and propose bash script.